### ISSUE TITLE
BUG EXIF BitmapHunter.java L 557

### ISSUE THREAD

**Issue by xX7** [square/picasso#1439]
Hey Guys, just wanted to let you know, that I found a bug in the current snapshot ( master branch ) in BitmapHunter.java line 557:

  int exifRotation = getExifRotation(exifOrientation); // exifRotation == 0
  int exifTranslation = getExifTranslation(exifOrientation); // exifTranslation == 1 // exifOrientation == 90
        if (exifRotation != 0) { <- NEVER reached
          matrix.preRotate(exifRotation);

The problem ist, that exifOrientation is already the value of the Rotation, in my case 90 Deg. 
getExifRotation on the other side is checking for the orientation value of the EXIF enum.
In other words : exifOrientation should be 6 instead of 90 for this code to work.

EDIT: The problem only occurs while selecting pictures from the Android Gallery. Pictures directly taken with the camera do not set the exifOrientation to the degreee value.
Camera Picture : exifOrientation = 6
Android Gallery : exifOrientation = 90
